.github/workflows/python_tests.yml

```diff
@@ -173,12 +174,12 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python: [3.6, 3.7, 3.8]
+        python: [3.6, 3.7, 3.8, 3.9]
         exclude:
           # TODO remove exclusion after issue with protobuf is solved
           # https://github.com/protocolbuffers/protobuf/issues/7765
           - os: windows-latest
-            python: 3.8
+            python: 3.9
```

Comment:
According the PR, this is solved for 3.9[Wheels are now present for Windows] IIUC. Can you check the PR and if it solved, can we remove the exclude for Python 3.9?